includes/ucf-vertical-slider-posttype.php

```diff
@@ -31,9 +31,15 @@ function vertical_slider_register_cpt() {
 			'menu_position' => 5,
 			'supports'      => array( 'title' ),
 			'has_archive'   => true,
+			'show_in_rest'  => true,
+			// 'rest_base'     => 'vertical-sliders',
 		);
 		register_post_type('vertical sliders', $args);
```

Comment:
Just seeing this from the previous pull request (and it explains why the rest api endpoint is odd) - custom post type names should be:

1. Lowercase
2. Singular (slider, not sliders)
3. Only contain letters and underscores
4. Should use underscores between each word.

Following these four rules, the custom post type should be registered as `vertical_slider`.

Reply:
```suggestion
		register_post_type('vertical_sliders', $args);
```